didcomm/did_doc/did_doc.py

```diff
@@ -90,3 +100,5 @@ class DIDCommService:
     service_endpoint: str
     routing_keys: List[DID_URL]
     accept: List[str]
+    # TODO make it non configurable (can it be another type?)
+    type: str = "DIDCommMessaging",
```

Comment:
Do we really need this field? Please note, that the class is created specifically for DIDComm services only.

Reply:
yes, during the deserialization when `CLS(**d)` happens

Reply:
removed